src/runtime/heap.rs

```diff
@@ -0,0 +1,16 @@
+use crate::*;
+
+pub unsafe fn heap_alloc(bytes: usize) -> RawPtr {
```

Comment:
Is this function actually unsafe? What ways could this be called which are unsafe? We should document safety explicitly.

Reply:
No you're right, this is safe.